.github/workflows/release.yml

```diff
@@ -97,18 +106,30 @@ jobs:
 
       - name: Install Ziti CI
         uses: openziti/ziti-ci@v1
+        with:
+          ziti-ci-version: latest
```

Comment:
Including the input is a reminder that it's necessary to override the default Git ref name, "latest," when testing `ziti-ci`.

Reply:
Pending required changes in https://github.com/openziti/ziti-ci/pull/59